composer.json

```diff
@@ -18,7 +18,7 @@
     },
     "require": {
         "php": ">=8.0",
-        "utopia-php/database": "0.38.*"
+        "utopia-php/database": "0.39.*"
```

Comment:
The lock file should be regenerated too.

Reply:
yes just fixed that.